flyteidl/clients/go/admin/pkce/auth_flow_orchestrator.go

```diff
@@ -68,7 +67,7 @@ func (f TokenOrchestrator) FetchTokenFromAuthFlow(ctx context.Context) (*oauth2.
 	// Pass along http client used in oauth2
 	httpClient, ok := ctx.Value(oauth2.HTTPClient).(*http.Client)
 	if !ok {
-		return nil, errors.New("Unable to retrieve httpClient used in oauth2 from context")
+		httpClient = http.DefaultClient
```

Comment:
can we leave a log saying that the default client is going to be used?

Reply:
Done